templates/base.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE "html">
```

Comment:
The DOCTYPE declaration should be `<!DOCTYPE html>` instead of `<!DOCTYPE "html">`. The quotes around `html` are incorrect and should be removed.